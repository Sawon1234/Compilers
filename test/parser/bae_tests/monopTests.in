fun testNegative(Integer a): Integer
{
	return -a;
}

fun testNegate(Boolean a) : Boolean
{
	return !a;
}