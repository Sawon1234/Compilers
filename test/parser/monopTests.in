fun testNegative( a : Integer): Integer
{
	return -a;
}

fun testNegate( a : Boolean) : Boolean
{
	return !a;
} return 0;