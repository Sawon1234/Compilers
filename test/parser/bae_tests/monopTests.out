fun testNegative < > ( a : Integer < > ) : Integer < > return a . negative ( ) ; fun testNegative < > ( a : Boolean < > ) : Boolean < > return a . negate ( ) ; 